streaming/java/streaming-runtime/src/test/java/org/ray/streaming/runtime/TestHelper.java

```diff
@@ -0,0 +1,18 @@
+package org.ray.streaming.runtime;
+
+public class TestHelper {
+
+  private static final String UT_PATTERN = "UT_PATTERN";
+
+  public static void setUTPattern() {
+    System.setProperty(UT_PATTERN, "");
+  }
+
+  public static void clearUTPattern() {
+    System.clearProperty(UT_PATTERN);
+  }
+
+  public static boolean isUTPattern() {
```

Comment:
Do you want to use this to tell if we are currently running tests?
What about just `isTest`? `ut pattern` is unclear.

Also, we don't need to set properties here. Just use a boolean flag.

Reply:
Fixed.